chromedriver_autoinstaller/utils.py

```diff
@@ -163,12 +163,16 @@ def get_chrome_version():
     elif platform == "win":
         # check both of Program Files and Program Files (x86).
         # if the version isn't found on both of them, version is an empty string.
-        dirs = [f.name for f in os.scandir("C:\\Program Files\\Google\\Chrome\\Application") if f.is_dir() and re.match("^[0-9.]+$", f.name)]
-        if dirs:
+        PROGRAMFILES = f"{os.environ['PROGRAMFILES']}\\Google\\Chrome\\Application"
+        PROGRAMFILESX86 = f"{os.environ['PROGRAMFILES(X86)']}\\Google\\Chrome\\Application"
+        
+        path = PROGRAMFILES if os.path.exists(PROGRAMFILES) else PROGRAMFILESX86 if os.path.exists(PROGRAMFILESX86) else None
+
+        if path:
+            dirs = [f.name for f in os.scandir(path) if f.is_dir() and re.match("^[0-9.]+$", f.name)]
             version = max(dirs)
         else:
-            dirs = [f.name for f in os.scandir("C:\\Program Files (x86)\\Google\\Chrome\\Application") if f.is_dir() and re.match("^[0-9.]+$", f.name)]
-            version = max(dirs) if dirs else ''
+            version = ''
```

Comment:
```suggestion
            # Could not find an installed chrome version.
            return
```
To be consistent with the other else below in this function .